### DISCUSSION TITLE
Globaleaks server hostname

### DISCUSSION THREAD

**Discussion by nicuznicuz** [globaleaks/globaleaks-whistleblowing-software#3558] [Q&A]
Hi everybody. Probably after I applied the last updates, my reverse-proxied globaleaks server is not accessible fron the outside, as it redirect all https traffic to its local ip address. As long as I remember, the server address is set via Network-HTTPS-hostname, but probably I don't recall correctly. Can anybody please give me a hint? Thank you in advance
Ciao
Nico

**Comment by gianlucagilardi** [comment]
Did you check SNI and SSL passthrough in your reverse proxy configuration?

**Comment by nicuznicuz** [comment]
As every user says "I did not touch anything" :-)

Seriously, even if I contact the server from the inside network using its
hostname, the http answer redirects to the server numeric ip address.

Il Mar 25 Lug 2023, 15:27 Gianluca Gilardi ***@***.***> ha
scritto:
